### PR TITLE
Fix issue #113 with updated local docker-compose

### PR DESCRIPTION
**Pull request by andrejpk**
# Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Checklist:

- [x] All new and existing tests passed.
- [x] The code follows the code style and conventions of this project.
- [ ] The change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

# Description of the change
Fix the relationship between the `iothubmanager` and `auth ` microservices by making changes in the docker-compose file for local deployment.
...

# Motivation for the change
When I pull the current version of this repo and attempt to follow the [Azure docs for the IOT Remote Monitoring solution accelerator](https://docs.microsoft.com/en-us/azure/iot-accelerators/iot-accelerators-remote-monitoring-deploy-local), I get similar errors like posted in issue #113 . In the container logs, the `iotmanager` container is failing, complaining about a missing `PCS_AUTH_WEBSERVICE_URL` parameter: 

```
iothubmanager_1     | [WebService.48a0b63c-c709-4c0e-8d56-026b04b41e7a][2018-08-01 17:34:42Z][ERROR][ConfigData:ProcessMandatoryPlaceholders] Environment variables not found, {"varsNotFound":", PCS_AUTH_WEBSERVICE_URL"}
iothubmanager_1     |
iothubmanager_1     | Unhandled Exception: Microsoft.Azure.IoTSolutions.IotHubManager.Services.Exceptions.InvalidConfigurationException: Environment variables not found: , PCS_AUTH_WEBSERVICE_URL
iothubmanager_1     |    at Microsoft.Azure.IoTSolutions.IotHubManager.Services.Runtime.ConfigData.ProcessMandatoryPlaceholders(String& value) in /home/azureiotpcs/vsts-agent/_work/14/s/iothub-manager/Services/Runtime/ConfigData.cs:line 109
iothubmanager_1     |    at Microsoft.Azure.IoTSolutions.IotHubManager.Services.Runtime.ConfigData.ReplaceEnvironmentVariables(String& value, String defaultValue) in /home/azureiotpcs/vsts-agent/_work/14/s/iothub-manager/Services/Runtime/ConfigData.cs:line 76
iothubmanager_1     |    at Microsoft.Azure.IoTSolutions.IotHubManager.Services.Runtime.ConfigData.GetString(String key, String defaultValue) in /home/azureiotpcs/vsts-agent/_work/14/s/iothub-manager/Services/Runtime/ConfigData.cs:line 42
iothubmanager_1     |    at Microsoft.Azure.IoTSolutions.IotHubManager.WebService.Runtime.Config..ctor(IConfigData configData) in /home/azureiotpcs/vsts-agent/_work/14/s/iothub-manager/WebService/Runtime/Config.cs:line 75
iothubmanager_1     |    at Microsoft.Azure.IoTSolutions.IotHubManager.WebService.Program.Main(String[] args) in /home/azureiotpcs/vsts-agent/_work/14/s/iothub-manager/WebService/Program.cs:line 22
iothubmanager_1     | /app/run.sh: line 6:     6 Aborted                 dotnet Microsoft.Azure.IoTSolutions.IotHubManager.WebService.dll
```
Once this container dies, the wheels come off and nothing seems to work. 
...

### CODE REVIEW

**Comment by ppathan** [comment]
@zhang-hua Can you please review this PR?

File: scripts/local/docker-compose.yml [hunk 1]
```diff
@@ -63,6 +64,7 @@ services:
       - PCS_AUTH_REQUIRED
       - PCS_CORS_WHITELIST
       - PCS_APPLICATION_SECRET
+      - PCS_AUTH_WEBSERVICE_URL=http://auth:9001/v1
```
Comment (zhang-hua) [inline scripts/local/docker-compose.yml]: nit: move this environment up before PCS_AUTH_ISSUER

File: scripts/local/docker-compose.yml [hunk 2]
```diff
@@ -54,7 +54,8 @@ services:
   iothubmanager:
     image: azureiotpcs/iothub-manager-dotnet:testing
     depends_on:
-      - storageadapter
+      - storageadapter      
+      - auth
```
Comment (zhang-hua) [inline scripts/local/docker-compose.yml]: nit: move it up

File: scripts/local/docker-compose.yml [hunk 3]
```diff
@@ -8,7 +8,7 @@
 # - note how this file references many environment variables, either set
 #   them in the .env file or edit this file adding the values here
 # - run `docker-compose up`
-# - open the browser at http://127.0.0.1:10443
+# - open the browser at http://127.0.0.1:8080
```
Comment (zhang-hua) [inline scripts/local/docker-compose.yml]: Good catch!

**Comment by zhang-hua** [comment]
@andrejpk , thanks for your contribution. We introduced application roles and authorization feature recently which will add service dependency on auth service for iothub-manager, config and telemetry micro services. So you still need similar changes for the other two micro services in the yaml files.

**Comment by andrejpk** [comment]
Thanks @zhang-hua .. should I add those dependencies and include them in this PR or do you already have this in flight elsewhere?

**Pull request merged**